doc/releases/release-notes-2.1.rst

```diff
@@ -17,53 +17,304 @@ Kernel
+20341 : drivers: pwm: add PWM shell : henrikbrixandersen : Henrik Brix Andersen
+19804 : drivers: sensor: Add OPT3001 light sensor driver : iosifmacesanu : Iosif Macesanu
+19805 : drivers: sensor: Add SI7060 temperature sensor driver : iosifmacesanu : Iosif Macesanu
+19161 : drivers/sensor: sht3xd: implement single shot mode : oleszczyk : None
+19749 : drivers: serial: Configurable GPIO management in NRFX UARTE : dwildmark : Dennis Wildmark
+19999 : drivers: serial: liteuart: Remove unused UART_LITEUART_PORT_0 symbol : ulfalizer : Ulf Magnusson
+19911 : drivers: spi: Add simple_spi driver : olofk : Olof Kindgren
+20075 : drivers : spi : mec1501 : XEC SPI driver : scottwcpg : Scott Worley
+19945 : drivers: spi_nor: Remove unused SPI_NOR_{PAGE,SECTOR}_SIZE symbols : ulfalizer : Ulf Magnusson
+17383 : drivers: spi_nor: support Microchip SPI flash global unblock : fabled : Timo Teräs
+18557 : drivers: spi: rv32m1: Added LPSPI driver for RV32M1 : karstenkoenig : Karsten König
+18817 : drivers: ssd1306: Write data from any coordinate : lgl88911 : Frank Li
+17398 : drivers: ssd16xx: rework (use bytestring property for LUT and registers) : jfischer-phytec-iot : Johann Fischer
+19091 : drivers/timer/hpet.c: migrate to devicetree : gnuless : Charles E. Youse
+18833 : drivers: usb_dc_sam: validate pointer in usb_dc_ep_is_stalled : jfischer-phytec-iot : Johann Fischer
+18175 : drivers: watchdog: Pass timeout in ms if iwdg is started at boot : aakkds : None
+17562 : drivers: watchdog: stm32: Support System Window Watchdog (WWDG) : gon1332 : Ioannis Konstantelias
+20290 : driver: uart: make deprecation effective : pabigot : Peter A. Bigot
+
+19944 : sensor: bmg160: Use BMG160_THREAD_PRIORITY instead of ignoring it : ulfalizer : Ulf Magnusson
+20347 : sensor: ccs811: Convert GPIOs to device tree : galak : Kumar Gala
+20413 : sensor: dts - convert several sensor drivers to DTS : galak : Kumar Gala
+20432 : sensor: hmc5883l: convert to device tree : microbuilder : Kevin Townsend
+19068 : sensor:lis2mdl: remove '-magn' extension from lis2mdl name : avisconti : Armando Visconti
+20343 : sensor: lis3mdl-magn: Convert GPIO INT to device tree : galak : Kumar Gala
+20350 : sensor: nrf5: Convert name to come from device tree : galak : Kumar Gala
+19624 : sensors: lis2dh: disable SDO/SA0 pull-up in case of LIS3DH : mniestroj : Marcin Niestroj
+20345 : sensor: vl53l0x: Convert GPIO XSHUT to device tree : galak : Kumar Gala
+
+20054 : serial: sam0: Add support for SAME54 : benpicco : None
+
+20475 : coccinelle: cleanups related to timeout : pabigot : Peter A. Bigot
+19650 : coccinelle: final pass standardizing integer timeout parameters : pabigot : Peter A. Bigot
+19623 : coccinelle: improve integer timeout script and fix missed cases : pabigot : Peter A. Bigot
+19562 : coccinelle: standardize kernel timeout arguments : pabigot : Peter A. Bigot
+
+
 Networking
 **********
 
-* Added new TCP stack implementation. The new TCP stack is still experimental
```

Comment:
You removed a bunch stuff..

Reply:
yeah, I realized after that the content was not just temp filler.